llvm/test/Analysis/LoopAccessAnalysis/scalable-vector-regression-tests.ll

```diff
@@ -24,7 +24,7 @@ loop.end:
   ret void
 }
 
-; CHECK-LABEL: LAA: Found a loop in regression_test_loop_access_scalable_typesize
+; CHECK-LABEL: 'regression_test_loop_access_scalable_typesize'
 ; CHECK: LAA: Bad stride - Scalable object:
```

Comment:
Should we be checking any extra lines after the label now?

Reply:
Fixed, thanks.